src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -93,13 +118,28 @@ export function generateOqlDisplayValue(oqlData: {[oqlLine: string]: IOqlData},
         const datum = oqlData[oqlLine];
 
         if (datum) {
-            oqlValue = datum.sequenced ? generateOqlValue(oqlData[oqlLine]) : "N/P";
+            oqlValue = datum.sequenced ? generateOqlValue(oqlData[oqlLine], alterationType) : "no alteration";
```

Comment:
I find it a little confusing to call this "oql" since its not.. maybe something like "alteration summary". or pseudoOqlSummary or something. Just makes it more readable IMO